doc/classes/CanvasItem.xml

```diff
@@ -648,7 +648,7 @@
 		</signal>
 		<signal name="hidden">
 			<description>
-				Emitted when the [CanvasItem] becomes hidden, either because its own [member visible] property was set to [code]false[/code] or the [member visible] property of one of its [CanvasItem] ancestors was set to [code]false[/code].
+				Emitted when the [CanvasItem] is hidden, i.e. no longer visible in the tree (see [method is_visible_in_tree]).
```

Comment:
Seems I missed this in the suggestion (https://github.com/godotengine/godot/pull/97367#discussion_r1771921358), I meant the same as what @Mickeon suggested but with `i.e.` instead of `that is`:
```suggestion
				Emitted when the [CanvasItem] is hidden, i.e. it's no longer visible in the tree (see [method is_visible_in_tree]).
```

Reply:
Ah yup. Fixed!